### PR TITLE
Add LMP date minimum validation for voice registration

### PR DESCRIPTION
**Pull request by rudigiesler**
This is the same as https://github.com/praekelt/mama-ng-jsbox/pull/164 , but for the voice registration app.

### CODE REVIEW

File: src/voice_registration.js [hunk 1]
```diff
@@ -359,12 +359,27 @@ go.app = function() {
...
         });
 
+        // Validation for minimum weeks for pregnancy
+        self.add('state_validate_lmp_date', function(name, creator_opts) {
+            var date = self.im.user.get_answer('working_date');
+
+            var weeks = go.utils_project.get_weeks_until_today(
+                self.im.config, date, 'YYYYMMDD');
+
+            console.log(weeks);
```
Comment (erikh360) [inline src/voice_registration.js]: Probably shouldn't be here...

**Review by KaitCrawford — APPROVED** [review]
Looks good to me :+1:

**Review by erikh360 — COMMENTED** [review]
Good just remove the console.log

File: src/voice_registration.js (hunk 1 above)
Comment (rudigiesler) [reply src/voice_registration.js]: Whoops, good catch

**Comment by rudigiesler** [comment]
@erikh360 `console.log` removed. Ready for another look.

**Comment by erikh360** [comment]
@rudigiesler there is another one, think you just need to run grunt before committing.

**Comment by rudigiesler** [comment]
@erikh360 Yup :man_facepalming: Forgot to build. Should be right now.

**Review by erikh360 — APPROVED** [review]

**Pull request merged**